.gitignore

```diff
@@ -252,3 +252,5 @@ tags
 
 
 # End of https://www.gitignore.io/api/vim,linux,emacs,macos,python,sublimetext
+
+telepresence.log
```

Comment:
This is conspicuous, add it before the gitignore.io section :wink:

Reply:
why? You mean on top of the file?